cumulusci.properties

```diff
@@ -1,4 +1,4 @@
-cumulusci.package.name=HEDAP
+cumulusci.package.name=HEDA
```

Comment:
Hey, @jlantz ~ This combined with the change to package.xml below should be ok, right? I figured I'd update it if it's feasible, now that we are starting from scratch.